docs/DMS/DMS-AUTH.md

```diff
@@ -37,62 +37,89 @@ update is authorized.
 
 ### 1. Retrieve Api Client Identity from Token
 
-When an API request comes in, the system extracts the identity of the API client by decoding the token included in the request.
+When an API request comes in, the system extracts the identity of the API client
+by decoding the token included in the request.
 
 ### 2. Retrieve associated Claimset
 
-The associated claimset for the Api client will be retrieved. This claimset defines Resource claims, Resource claims actions, Resource claims action authorization strategies.
+The associated claimset for the Api client will be retrieved. This claimset
+defines Resource claims, Resource claims actions, Resource claims action
+authorization strategies.
 
 ### 3. Retrieve and cache Basic Claims details
 
-The system retrieves additional basic claims details, such as the education organization id, key, profiles, and namespace prefixes for the ApiClient. These details are stored in a cached context.
+The system retrieves additional basic claims details, such as the education
+organization id, key, profiles, and namespace prefixes for the ApiClient. These
+details are stored in a cached context.
 
 ### 4. Retrieve Claimset details from Security Database (Cached)
 
-The claimset details, including Resource claims, Resource claims actions, Resource claims action authorization strategies, are retrieved from a security database.
+The claimset details, including Resource claims, Resource claims actions,
+Resource claims action authorization strategies, are retrieved from a security
+database.
```

Comment:
We can be more specific: the DMS will retrieve this information from the DMS Configuration Service, which will store the data in its own database.